src/Stratis.Bitcoin/Consensus/ConsensusManager.cs

```diff
@@ -284,7 +237,7 @@ private void PeerDisconnectedLocked(int peerId)
         /// <summary>
         /// A callback that is triggered when a block that <see cref="ConsensusManager"/> requested was downloaded.
         /// </summary>
-        private void ProcessDownloadedBlock(ChainedHeaderBlock chainedHeaderBlock)
+        private void OnBlockDownloaded(ChainedHeaderBlock chainedHeaderBlock)
```

Comment:
Same as for other ones: it's not a callback, please return the old name

Reply:
This has been reverted.